argilla-frontend/components/base/base-breadcrumbs/BaseBreadcrumbs.vue

```diff
@@ -29,11 +29,12 @@
           v-else
           class="breadcrumbs__item --action"
           @click="onBreadcrumbAction(breadcrumb)"
+          :aria-current="breadcrumb.name"
           >{{ breadcrumb.name }}</span
         >
       </li>
     </ul>
```

Comment:
Consider providing a more descriptive comment about this component's functionality.